common/event_modifiers/wc_event_modifiers.txt

```diff
@@ -1628,4 +1628,13 @@ barrow_den_average_protection = {
 barrow_den_good_protection = {
 	icon = 5
 	defensive_plot_power_modifier = 0.5
+}
+ulduar_closed_down = {
+	icon = 82
+	disease_defence = 10
+	garrison_size = 25
+	levy_size = -1.3
```

Comment:
Doesn't mean that you can't get levy from Ulduar

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/67056313-2f064e00-f154-11e9-92a5-276f70abbdba.png)

</details>

Reply:
thats ok since its small